### DISCUSSION TITLE
Can't install DsHidMini on Windows 7

### DISCUSSION THREAD

**Discussion by Phantasm90** [nefarius/DsHidMini#162] [Support]
I tried to install the latest release and every time i get "The .INF file you selected does not support this method of installation." What can i do? I previously had Scp Toolkit but i believe i uninstalled everything

**Comment by nefarius** [comment]
It clearly as day states in the README that Windows 7 is not supported.

**Reply by B3bomber** [reply]
I managed to get it to install on Win 7 x64, though obviously with "missing libraries" it does not function as intended. It is recognized as a driver but it does not know what the driver is for. I mean that as in the popup for the are you sure you want to install this driver? came up and everything. The right click menu does not work as stated by the guy above.

I rarely offer myself as a beta tester but I will for this. You interested?

**Reply by nefarius** [reply]
*Sigh* OK here it is one more time: neither DMF (a mandatory dependency) nor UMDF (the framework this project uses) is available for Windows 7. So no, I don't need beta testers for something that is not possible. Period.

**Reply by B3bomber** [reply]
That's a shame. Thanks for the reply at least.

I remember when Microsoft didn't intentionally try to nuke backwards compatability (it's not actually a new framework).

**Reply by nefarius** [reply]
I agree there's little technical reason to not backport at least the compatible sections of the frameworks to Windows 7 but I assume it's a management/political decision internally so people aren't even allowed to work on legacy OS topics anymore for the glory of Windows 10, the last Windows ever! Oh wait, I mean Windows 11, which is just 10 with a reskin. Can't wait for Windows 12, the last Windows version ever!
